### DISCUSSION TITLE
Install IMS apk for NOT supported

### DISCUSSION THREAD

**Discussion by DistroAgnostic** [phhusson/treble_experimentations#2423] [Q&A]
Device manufacturer: Samsung
Model: Galaxy M21(Exynos 9611)
I cannot make any calls on my phone as it says "Install IMS apk for NOT supported" (maybe because it's Exynos, not snapdragon). I also downloaded the IMS service of Samsung from apk mirror but it is not installing says conflicts with the previous IMS package.
Whenever I call it says network busy. Please help.
Also, I installed the same gsi on my tablet and from phh settings I was able to install IMS as my tablet has a snapdragon processor and I was able to make calls but another person is not able to hear me. Tablet model: Lenovo YT-X705X
![photo_2022-08-28_10-37-06](https://user-images.githubusercontent.com/98891600/187058890-302d06fc-5a62-4af4-b179-2e150097fbdf.jpg)

**Comment by code-fanatic** [comment]
I have call sms and internet working perfectly
Just VoLTE is not available

**Comment by DistroAgnostic** [comment]
Yeah, airtel works.
